test/integration/targets/wait_for_connection/aliases

```diff
@@ -0,0 +1,2 @@
+posix/ci/group2
+windows/ci/group2
```

Comment:
PEP 8 requires a newline at the end of python files. Perhaps you're confusing it with a blank line?

As for the rational, it's about making clean diffs for future PRs. If I need to an another line at the end of the file, I need to modify the last line to add a newline. If it already has a newline, that's not necessary.